src/java.xml/share/conf/jaxp-strict.properties

```diff
@@ -0,0 +1,120 @@
+################################################################################
+#           JAXP Strict Configuration File
+#
+# This property file, jaxp-strict.properties, represents a more secure configuration
+# for XML processing. It provides settings that are more restrictive than the
+# current default jaxp.properties. In particular:
+#   - JDKCatalog Resolve is on "strict" setting
+#   - Extension Functions are disabled
+#   - JAXP Limits are set to smaller numbers
+#
+# This configuration file can be set up using the system property
+# java.xml.config.file to override the default configuration jaxp.properties
+# and used to assess the impact of a stricter configuration, for example:
+#
+#     java -Djava.xml.config.file=$JAVA_HOME/conf/jaxp-strict.properties
+#
+# The pathname to the configuration file must be valid. If it is not absolute,
```

Comment:
I think it would be better to drop this paragraph or else just replace it with a sentence to say that the java.xml module description specifies the system property.

Reply:
Thanks Alan. Replaced the paragraph with a reference to the module description.